AUTHORS

```diff
@@ -0,0 +1 @@
+2019, Alberto Navalon Lillo <albertonl.dev@gmail.com>
```

Comment:
maybe also add the other two committers.

Reply:
Yes, sure